### ISSUE TITLE
Controlled-duration bans

### ISSUE THREAD

**Issue by zorbathut** [themotte/tildes#27]
Along with "ban permanently", we could really use "ban for a given time period".

**Comment by JulianRota** [comment]
Note that the `User` model already has [`ban_expiry_time`](https://github.com/themotte/tildes/blob/master/tildes/tildes/models/user/user.py#L125), and it appears to be enforced around the codebase. The UI for this should set that field to the desired time.

**Comment by Deimos** [comment]
Yes, temp bans are functional, but there's no interface for them currently. I've only ever used them a couple of times, and I just set that column on the user's row manually. There's a cronjob that runs hourly that will lift any bans that have expired.

**Comment by zorbathut** [comment]
Copied from another post, note that there's already a minimal ban interface:

https://github.com/themotte/tildes/blob/master/tildes/tildes/templates/user.jinja2#L192-L205

and so presumably this just needs to be augmented with an optional "timed ban" button and wired into the database.
